packages/spammer/spammer.go

```diff
@@ -61,6 +62,9 @@ func (s *Spammer) run(rate int, timeUnit time.Duration, imif string) {
 			return
 		}
 
+		estimatedDuration := messagelayer.Tangle().RateSetter.Estimate()
```

Comment:
We should avoid importing plugins in packages,
maybe make this estimation function a parameter of `run()`?